docs/_posts/Mary-Sci/2023-08-04-legclf_parameter_efficient_prediction_en.md

```diff
@@ -0,0 +1,119 @@
+---
+layout: model
+title: Legal Parameter Efficiency Prediction in Domain-Specific Documents
+author: John Snow Labs
+name: legclf_parameter_efficient_prediction
+date: 2023-08-04
+tags: [en, licensed, classification, legal, tensorflow]
+task: Text Classification
+language: en
+edition: Legal NLP 1.0.0
+spark_version: 3.0
+supported: true
+engine: tensorflow
+annotator: LegalBertForSequenceClassification
+article_header:
+type: cover
+use_language_switcher: "Python-Scala-Java"
+---
+
+## Description
+
+This is a multiclass classification model trained to estimate parameter efficiency in legal domain-specific documents. The model demonstrates remarkable proficiency in predicting `business`, `constitutional-law`, `contract-law`, `copyright`, `criminal-law`, `employment`, `liability`, `privacy`, `tax-law`, `trademark` classes for a wide variety of legal issues.
```

Comment:
Change the description to clarify that we are classifying stack exchange questions into topics. For example:

> This is a multiclass classification model trained on the Law Stack Exchange dataset that classifies law-related questions into one of the following categories:  `business`, `constitutional-law`, `contract-law`, `copyright`, `criminal-law`, `employment`, `liability`, `privacy`, `tax-law`, `trademark`.